app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,139 @@
+class OrganizationsController < ApplicationController
+  def new
+    authorize Organization, :new?
+  end
+  def create
+    authorize Organization, :create?
+    organization_name = params[:organization_name]
+    existing_organization = Organization.where("LOWER(name) = ?", organization_name.downcase).first
```

Comment:
Can you check if this works or not?
```suggestion
    existing_organization = Organization.find_by(name: organization_name.downcase)
```

Reply:
I checked & it worked bhaiya